src/math/Interpolant.js

```diff
@@ -47,7 +47,7 @@ Interpolant.prototype = {
 
 		validate_interval: {
 
-			seek: {
+			seek: while(true) {
```

Comment:
Unfortunately I haven't been able to fully understand this piece of code yet so I can't predict the side effects of this change.

Hopefully @tschw can find some time to double check and give green light on this one.